pom.xml

```diff
@@ -2190,6 +2190,7 @@
                 <spark.binary.version>3.2</spark.binary.version>
                 <delta.artifact>delta-core</delta.artifact>
                 <delta.version>2.0.2</delta.version>
+                <iceberg.version>1.4.3</iceberg.version>
```

Comment:
Iceberg 1.5.0 removed support for Spark 3.2. https://github.com/apache/iceberg/pull/9295

Reply:
let's leave a comment in `pom.xml`
```
<--! Iceberg 1.5.0 removed support for Spark 3.2. apache/iceberg#9295 -->
```